source/localizable/tutorial/service.md

```diff
@@ -308,3 +308,35 @@ export default Ember.Object.extend({
 We should now see some end to end maps functionality show up on our front page!
 
 ![super rentals homepage with maps](../../images/service/style-super-rentals-maps.png)
+
+### Stubbing Services in Acceptance Tests
+
+Finally, we want to update our acceptance tests to account for our new service.
+While it would be great to verify that a map is displaying, we don't want hammer the Google Maps API every time we run our acceptance test, 
+so for this tutorial we'll rely on our component integration tests to ensure that the map dom is being attached to our screen.
+To avoid hitting our maps request limit, we'll stub out our Maps service in our acceptance tests.
+
+Often, services connect to third party services that are not desirable to include in automated tests.
```

Comment:
Two uses of services might be confusing, maybe change 'third party services' to 'third party APIs'